xilem_web/web_examples/elm/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "elm"
+version = "0.0.0" # not versioned
+publish = false
+license.workspace = true
+edition.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+console_error_panic_hook = "0.1.7"
+console_log = { version = "1.0.0", features = ["color"] }
+log = "0.4.22"
+xilem_web = { path = "../.." }
+xilem_core = { workspace = true }
```

Comment:
I don't think this dependency is necessary here, you can just use `xilem_web::core` which is a reexport of `xilem_core`